server/src/RokuDevice.ts

```diff
@@ -38,7 +39,7 @@ export class RokuDevice {
 	 */
 	public async getScreenshot(outputFilePath: string) {
 		await this.generateScreenshot();
-		await this.saveScreenshot(outputFilePath);
+		return await this.saveScreenshot(outputFilePath);
```

Comment:
Return await is redundant (unless contained in a try/catch). [Info here](https://jakearchibald.com/2017/await-vs-return-vs-return-await/)